src/coreComponents/physicsSolvers/simplePDE/PhaseFieldDamageFEMKernels.hpp

```diff
@@ -210,36 +204,44 @@ class PhaseFieldDamageKernel :
     real64 qp_grad_damage[3] = {0, 0, 0};
     FE_TYPE::valueAndGradient( N, dNdX, stack.nodalDamageLocal, qp_damage, qp_grad_damage );
 
+    real64 D = 0;                                                                   //max between threshold and
+                                                                                    // Elastic energy
+    if( m_localDissipationOption == 1 )
+    {
+      D = fmax( threshold, strainEnergyDensity );
+    }
+
     for( localIndex a = 0; a < numNodesPerElem; ++a )
     {
       if( m_localDissipationOption == 1 )
       {
-        stack.localResidual[ a ] += detJ * ( -3 * N[a] / 16  -
-                                             0.375*pow( ell, 2 ) * LvArray::tensorOps::AiBi< 3 >( qp_grad_damage, dNdX[a] ) -
-                                             (0.5 * ell * D/Gc) * N[a] * m_constitutiveUpdate.getDegradationDerivative( qp_damage ));
+        stack.localResidual[ a ] += detJ * ( -3 * N[a] / 16
+                                             - 0.375*pow( ell, 2 ) * LvArray::tensorOps::AiBi< 3 >( qp_grad_damage, dNdX[a] )
+                                             - (0.5 * ell * D/Gc) * m_constitutiveUpdate.getDegradationDerivative( qp_damage ) * N[a]
+                                             - 0.5 * ell * extDrivingForce/Gc * N[a] );
       }
       else
       {
-        stack.localResidual[ a ] -= detJ * ( N[a] * qp_damage +
-                                             (pow( ell, 2 ) * LvArray::tensorOps::AiBi< 3 >( qp_grad_damage, dNdX[a] ) +
-                                              N[a] * m_constitutiveUpdate.getDegradationDerivative( qp_damage ) * (ell*strainEnergyDensity/Gc)) );
+        stack.localResidual[ a ] -= detJ * ( N[a] * qp_damage
+                                             + (pow( ell, 2 ) * LvArray::tensorOps::AiBi< 3 >( qp_grad_damage, dNdX[a] )
+                                                + N[a] * (ell*strainEnergyDensity/Gc) * m_constitutiveUpdate.getDegradationDerivative( qp_damage ) )
+                                             + ell * extDrivingForce/Gc * N[a] );
 
       }
       for( localIndex b = 0; b < numNodesPerElem; ++b )
       {
         if( m_localDissipationOption == 1 )
         {
           stack.localJacobian[ a ][ b ] -= detJ *
```

Comment:
do you need the case without dissipation?

Reply:
> do you need the case without dissipation?

Actually no, as Kumar's model is for the case with `m_localDissipationOption == 1`. I have removed the extDrivingForce in the case when `m_localDissipationOption != 1`